docs/bos/queryapi/how-works.md

```diff
@@ -0,0 +1,45 @@
+---
+id: how-it-works
+title: How QueryAPI works
+sidebar_label: How it works
+---
+
+QueryApi is a streaming indexer implementation that executes queries written by developers on the NEAR blockchain.
```

Comment:
I'm not sure `queries` is the best term here, as it might be confused with GraphQL queries? Best I can come up with is 'indexer code' but that doesn't seem great 😅

Reply:
I used `... executes custom indexing logic written ...`. Feel free to edit or change it 👍